### PR TITLE
[ng] toh-2: ng.io sync and elimination of snippets folder

### LINKED ISSUE #499: [ng] eliminates snippets folders from examples

### PR DESCRIPTION
**Pull request by chalin**
Fixes #499

### CODE REVIEW

File: src/angular/tutorial/toh-pt2.jade [hunk 1]
```diff
@@ -44,298 +39,263 @@ include ../_util-fns
...
+    When users select a hero from the list, the selected hero should appear in the details view.
+    This UI pattern is known as "master/detail."
+    In this case, the _master_ is the heroes list and the _detail_ is the selected hero.
 
-    ### Click event
+    Next you'll connect the master to the detail through a `selectedHero` component property,
+    which is bound to a click event.
 
-    We modify the `<li>` by inserting an Angular event binding to its click event.
+    ### Add a click event
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt2.jade]: Technically this is inaccurate. It should be:

### Add a click event binding
or
### Handle click events

File: src/angular/tutorial/toh-pt2.jade [hunk 2]
```diff
@@ -44,298 +39,263 @@ include ../_util-fns
...
 
-  ### What's next
+  ## The road ahead
 
-  Our Tour of Heroes has grown, but it’s far from complete.
-  We can't put the entire app into a single component.
-  We need to break it up into sub-components and teach them to work together
-  as we learn in the [next page](toh-pt3.html).
+  You've expanded the Tour of Heroes app, but it's far from complete.
+  You can't put the entire app into a single component.
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt2.jade]: "can't" isn't exactly right here. It's possible, isn't it? Maybe:

* An app shouldn't be one monolithic component.

File: src/angular/tutorial/toh-pt2.jade [hunk 3]
```diff
@@ -44,298 +39,263 @@ include ../_util-fns
...
+  In the [next page](toh-pt3.html), you'll split the app into sub-components and make them work together.
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt2.jade]: sub-components -> subcomponents

File: src/angular/tutorial/toh-pt2.jade [hunk 4]
```diff
@@ -44,298 +39,263 @@ include ../_util-fns
...
+
+  +makeExcerpt('app/app.component.1.html (template excerpt)', 'selectedHero-details')
 
   :marked
     ### Hide the empty detail with ngIf
 
-    When our app loads we see a list of heroes, but a hero is not selected.
-    The `selectedHero` is `undefined`.
-    That’s why we'll see the following error in the browser’s console:
+    When the app loads, the `selectedHero` is null and won't be initialized until you click a hero's name.
```
Comment (kwalrath) [inline src/angular/tutorial/toh-pt2.jade]: null and won't be ... click
->
null. The selected hero isn't initialized until the user clicks

**Comment by kwalrath** [comment]
Trying not to nitpick, but I couldn't help noticing a few. Could you stage this? (If not, no worries, I can do it later.)

File: src/angular/tutorial/toh-pt2.jade (hunk 1 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt2.jade]: Right. Fixed.

File: src/angular/tutorial/toh-pt2.jade (hunk 2 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt2.jade]: Right. Fixed.

File: src/angular/tutorial/toh-pt2.jade (hunk 3 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt2.jade]: fixed

File: src/angular/tutorial/toh-pt2.jade (hunk 4 above)
Comment (chalin) [inline src/angular/tutorial/toh-pt2.jade]: fixed

**Comment by chalin** [comment]
All good feedback, thanks.
Staged at https://angulardart-org-dev2.firebaseapp.com/angular/tutorial/toh-pt2

**Pull request merged**